### PR TITLE
Create View+ConditionalModifier.swift

### PR DESCRIPTION
**Pull request by alemannosara**
This PR adds a conditional modifier to a SwiftUI View.

### CODE REVIEW

File: Sources/Qit/Extensions/SwiftUI/View+ConditionalModifier.swift [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+import SwiftUI
+
+extension View {
+
+    typealias ContentTransform<Content: View> = (Self) -> Content
+
+    /// Conditional `SwiftUI` view modifier
+    /// - Parameters:
+    ///   - condition: VIew modifier condition
```
Comment (AlexNsbmr) [inline Sources/Qit/Extensions/SwiftUI/View+ConditionalModifier.swift]: I made a small typo here: 
```suggestion
    ///   - condition: View modifier condition
```

**Comment by AlexNsbmr** [comment]
@alemannosara it would be great to add some tests if possible :)

**Review by AlexNsbmr — APPROVED** [review]

**Comment by AlexNsbmr** [comment]
@alemannosara we should as well upgrade the xcode version and testing device for the CI https://github.com/qoqa/Qit/blob/master/.github/workflows/ci.yml

**Review by AlexNsbmr — COMMENTED** [review]
@alemannosara we should as well upgrade the xcode version and testing device for the CI https://github.com/qoqa/Qit/blob/master/.github/workflows/ci.yml

**Review by mpintoQoQa — APPROVED** [review]

**Pull request merged**